hyperspy/tests/signal/test_2D_tools.py

```diff
@@ -122,3 +124,16 @@ def test_align_expand(self):
         # Check alignment is correct
         d_al = s.data[:, ds[0]:-ds[0], ds[1]:-ds[1]]
         nt.assert_true(np.all(d_al == self.aligned))
+
+
+class TestPhaseFunctions:
+
+    def test_add_phase_ramp(self):
```

Comment:
This is testing `add_ramp`, not `add_phase_ramp`. Beside, as there is no `setUp` method, it'll be equivalent to turn the method into a function.